src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -409,19 +415,63 @@ private MyPublicationViewModel HydrateMyPublicationViewModel(Publication publica
                 releaseViewModel.Permissions = PermissionsUtils.GetReleasePermissions(_userService, release);
             });
 
+            publicationViewModel.Methodologies = await HydrateMethodologyVersionViewModels(publication);
```

Comment:
The `.Ignore` is actually due to not wanting to use AutoMapper to build the `MethodologyVersionViewModels` themselves, not specifically the permissions, although determining the permissions are part of the reason for not using it.

As well as the complexity of needing to use a permissions resolver class with AutoMapper to determine the permissions, using Automapper to map the `MethodologyVersion` itself needs to work out the latest version of the `PublicationMethodology` and it's clearer for this to happen in the service rather than be obscured in AutoMapper config.

I'm not sure it's any less efficient and when it comes to the cleanup we'll be removing the entire mapping from `Publication` to `MyPublicationViewModel` imminently anyway like you say.

I'm not sure how the `.Ignore` could be misunderstood but I've added a comment to explain why it's there now.